### ISSUE TITLE
min length of alignment -l 

### ISSUE THREAD

**Issue by shri1984** [ruanjue/wtdbg2#225]
Hey, 
I am trying to figure out if I use -l 5000, which is not a multiple of 256, then what number wtdbg2 is taking if it doesn't match multiple of 256. I am getting different results when I used -l 5000 and -l 4096.

**Comment by ruanjue** [comment]
You are right, will change it in future version. BTW, `-l 5000` was inherited from SMARTdenovo.
